lib/vcr/cassette/serializers/json.rb

```diff
@@ -3,7 +3,7 @@
 module VCR
   class Cassette
     class Serializers
-      # The JSON serializer. Uses `MultiJson` under the covers.
```

Comment:
This is no longer true. See https://github.com/vcr/vcr/pull/815.

Reply:
We would accept a tiny PR about removing it!

Reply:
👌 https://github.com/vcr/vcr/pull/841